Formula/step.rb

```diff
@@ -0,0 +1,45 @@
+class Step < Formula
+  desc "Crypto and x509 Swiss-Army-Knife"
+  homepage "https://smallstep.com"
+  url "https://github.com/smallstep/cli/releases/download/v0.8.3/brew_step_0.8.3.tar.gz"
+  sha256 "6f93906ff1bea17dad02e8509fd06351ff3d8bdaa921b46f36aacfa47c44c762"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/smallstep").install buildpath.children
+    cd "src/github.com/smallstep/cli" do
+      system "make", "build"
+      bin.install "bin/step" => "step"
+      bash_completion.install "autocomplete/bash_autocomplete" => "step"
+    end
+    cd "src/github.com/smallstep/certificates" do
+      system "make", "build"
+      bin.install "bin/step-ca" => "step-ca"
+    end
+  end
+
+  test do
+    # Generate a public / private key pair. Creates foo.pub and foo.priv.
+    system "#{bin}/step", "crypto", "keypair", "foo.pub", "foo.priv", "--no-password", "--insecure"
+    assert_equal "foo.priv\nfoo.pub\n", shell_output("ls foo*")
```

Comment:
Rather than shelling out, can you please use

```suggestion
     assert_predicate testpath/"foo.priv", :exist?
     assert_predicate testpath/"foo.pub", :exist?
```